### PR TITLE
Sprint12 staging

### PR DESCRIPTION
**Pull request by jeremythuff**
#86 - Weaver component's build process should build multiple versions of the product.
#102 - Weaver Usage Docs should be converted into an angular project.
#223 - The wvr manifest action to submit request should afford providing response map.
#232 - Handlebars should be extracted into a service and all helpers declared outside of the component constructor.
#233 - On safari (check firefox) mobile display does not run the position footer function at load time.
#234 - Weaver components should provide a wvr-cards component.
#237 - Weaver Component selectors need to be refactored to avoid name collision with Weaver (angularjs) directives.
#255 - The wvr-header should use the configured asset path for the header logo.

### CODE REVIEW

File: .wvr-ud/static-assets/styles.css [hunk 1]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
 .feather-plus-circle,
 .open .feather-minus-circle {
   display: inline-block;
-}
\ No newline at end of file
+}
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:151]: This should be broken down into partial files and imported.

File: .wvr-ud/static-assets/styles.css [hunk 2]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
...
+
+.feather-plus-circle,
+.open .feather-minus-circle {
+  display: inline-block;
+}
+
+/*** OVERRIDES ***/
+
+html {
+  height: 100%;
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:307]: This is a duplicated property with the same value on html.

File: .wvr-ud/static-assets/styles.css [hunk 3]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
...
+}
+body {
+  font-size: .875rem;
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:310]: These are duplicated properties with the same values on body.

File: .wvr-ud/static-assets/styles.css [hunk 4]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
...
+  height: 100%;
+  overflow: hidden;
+}
+
+.bi {
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:315]: This class is also duplicated with same values.

File: .wvr-ud/static-assets/styles.css [hunk 5]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
...
+.feather-minus-circle,
+.open .feather-plus-circle  {
+  display: none;
+}
+
+.feather-plus-circle,
+.open .feather-minus-circle {
+  display: inline-block;
+}
+html {
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:457]: Duplicate

File: .wvr-ud/static-assets/styles.css [hunk 6]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
...
+  height: 100%;
+}
+body {
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:460]: Duplicate

File: .wvr-ud/static-assets/styles.css [hunk 7]
```diff
@@ -148,4 +148,2292 @@ main.loading #preview {
...
+  font-size: .875rem;
+  height: 100%;
+  overflow: hidden;
+}
+
+.bi {
```
Comment (wwelling) [inline .wvr-ud/static-assets/styles.css:466]: Duplicate

**Review by wwelling — CHANGES_REQUESTED** [review]
Many duplicates in styles.css. It is also too large of a file and should be decomposed.

File: .wvr-ud/static-assets/styles.css (hunk 1 above)
Comment (jeremythuff) [reply .wvr-ud/static-assets/styles.css:151]: This was artificially long due to a bug. That bug has been removed.

File: .wvr-ud/static-assets/styles.css (hunk 2 above)
Comment (jeremythuff) [reply .wvr-ud/static-assets/styles.css]: See the above comment

File: .wvr-ud/static-assets/styles.css (hunk 3 above)
Comment (jeremythuff) [reply .wvr-ud/static-assets/styles.css]: See above

File: .wvr-ud/static-assets/styles.css (hunk 4 above)
Comment (jeremythuff) [reply .wvr-ud/static-assets/styles.css]: See above

File: .wvr-ud/static-assets/styles.css (hunk 5 above)
Comment (jeremythuff) [reply .wvr-ud/static-assets/styles.css]: See above

File: .wvr-ud/static-assets/styles.css (hunk 7 above)
Comment (jeremythuff) [reply .wvr-ud/static-assets/styles.css]: See above

File: projects/wvr-elements/package.json [hunk 8]
```diff
@@ -21,9 +21,11 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "css-element-queries": "^1.2.3"
-  ,    "tslib": "^2.0.0"
-},
+    "css-element-queries": "^1.2.3",    "tslib": "^2.0.0"
```
Comment (wwelling) [inline projects/wvr-elements/package.json:24]: Bad formatting

File: package.json [hunk 9]
```diff
@@ -4,16 +4,11 @@
...
   "license": "MIT",
-  "bin": {
-    "wvr-ud": "./scripts/build-wvr-ud.js",
-    "wvr-build-components": ".scripts/build-wvr-components.js",
-    "wvr-build-configuration": ".scripts/build-wvr-components-configuration.js",
-    "wvr-build-lighthouse": ".scripts/build-wvr-components-lighthouse-badges.js",
-    "wvr-build-static": ".scripts/build-wvr-components-static.js",
-    "wvr-serve-dist": ".scripts/serve-dist.js"
+  "config": {
+    "DOCKER_SERVER": "10.55.134.48/weaver/wvr-components"
```
Comment (wwelling) [inline package.json:11]: Is this something we want in a public repo?

File: package.json [hunk 10]
```diff
@@ -50,69 +51,79 @@
...
-    "@angular/common": "^10.0.12",
-    "@angular/compiler": "^10.0.12",
-    "@angular/core": "^10.0.12",
-    "@angular/elements": "^10.0.12",
-    "@angular/forms": "^10.0.12",
-    "@angular/localize": "^10.0.12",
-    "@angular/platform-browser": "^10.0.12",
-    "@angular/platform-browser-dynamic": "^10.0.12",
-    "@angular/router": "^10.0.12",
+    "-": "0.0.1",
```
Comment (wwelling) [inline package.json:54]: What is this?

**Review by wwelling — APPROVED** [review]
A few more minor comments.

**Pull request merged**